src/excmds.ts

```diff
@@ -2070,6 +2070,7 @@ export async function changelistjump() {
     const tail = state.prevInputs[state.prevInputs.length - 1]
     const jumppos = tail.jumppos ? tail.jumppos : state.prevInputs.length - 1
     const input = state.prevInputs[jumppos]
+    state.storeNextInput = "false"
```

Comment:
Should check if `input` is real before setting this

Reply:
Probably worth making a `setAsync` function so that we can wait for this to be successfully set.

https://github.com/tridactyl/tridactyl/blob/99941d796ea665c04accff61e2dea1e1bcb2ead0/src/state.ts#L100